ops/services/container_instances/db_rollback/db_rollback.tf

```diff
@@ -2,10 +2,9 @@ resource "azurerm_container_group" "db_rollback" {
   name                = "${var.name}-${var.env}-db-rollback"
   location            = var.resource_group_location
   resource_group_name = var.resource_group_name
-  ip_address_type     = "Private"
+  ip_address_type     = "None"
```

Comment:
Have we tested this change yet? Just want to make sure.

Reply:
This was tested, but I intend to test it again before rollout due to changes that have gone in since.